addons/mail/models/mail_thread.py

```diff
@@ -4640,6 +4640,8 @@ def _thread_to_store(self, store: Store, /, *, fields=None, request_list=None):
                 ]))
             if request_list and "suggestedRecipients" in request_list:
                 res["suggestedRecipients"] = thread._message_get_suggested_recipients()
+            if request_list and "userSignature" in request_list:
+                res["userSignature"] = self.env.user.signature
```

Comment:
Hello, is the user signature thread specific? It looks like this is not done in the right model.

I'd say it should be moved to `_init_store_data` (as an extra field of "self", for which the value is returned in partner `_to_store` when the field is requested).

Reply:
Nope, it is the field on user so indeed, not thread specific :)